content/en/core/releases/4.11.0.md

```diff
@@ -0,0 +1,106 @@
+
+---
+title: "4.11.0 release notes"
+linkTitle: "4.11.0"
+weight:
+description: >
+relevantLinks: >
+toc_hide: true
+---
+
+## Known issues
+
+Check the repository for the [latest known issues](https://github.com/medic/cht-core/issues?q=is%3Aissue+label%3A%22Affects%3A+4.11.0%22).
+
+## Upgrade notes
+
+### Breaking changes
+
+None.
+
+### UI/UX changes
+
+None.
+
+## Highlights
+
+### The CHT-Datasource API now supports querying persons
+
+The CHT datasource API has been expanded to allow querying places and persons by contact type
+
+##### Details
+see more details in the [api docs](http://docs.communityhealthtoolkit.org/cht-datasource/).
+
+### Phone numbers are not required to be checked for duplicates
+Phone number fields can now be used without automatically checking for duplicates. This can be useful for workflows where users may share a phone, or for any workflow where having duplicate phones is possible and its undesirable to check for duplicates.
```

Comment:
```suggestion
Phone number fields can now be used without automatically checking for duplicates. This can be useful for workflows where users may share a phone or for any workflow where having duplicate phones is possible, and it's undesirable to check for duplicates.
```